### PR TITLE
Al 37/swagger

### PR DESCRIPTION
**Pull request by HoangCacLoanChau**
## Jira link

https://capstone-alumni.atlassian.net/jira/software/c/projects/AL/boards/1?modal=detail&selectedIssue=AL-37
## Describe your changes

package.json: add swagger-ui-react package

## Screenshot/Record (FE)
image
image

## Checklist before requesting a review
1. [x] I have performed a self-review of my code
2. [x] Have you lint your code locally before submission?

### CODE REVIEW

File: src/app/api-doc/ReactSwagger.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+'use client';
```
Comment (congtrung6391) [inline src/app/api-doc/ReactSwagger.tsx:1]: could you rename `api-doc` to `api_doc`?

Comment (congtrung6391) [reply src/app/api-doc/ReactSwagger.tsx:1]: Why do we need to split this component?

File: src/pages/api/signIn.ts [hunk 2]
```diff
@@ -2,6 +2,29 @@ import { NextApiResponse } from 'next';
 import { NextApiRequest } from 'next';
 import SessionController from 'src/modules/sessions/controller/session.controller';
 
+
+/**
+ * @swagger
```
Comment (congtrung6391) [inline src/pages/api/signIn.ts:7]: Could you update `README.md` to show us how to use this lib? Or you could write some note in https://capstone-alumni.atlassian.net/wiki/spaces/AC/pages/393511/Technical+Investigation

File: src/app/api-doc/ReactSwagger.tsx (hunk 1 above)
Comment (HoangCacLoanChau) [reply src/app/api-doc/ReactSwagger.tsx:1]: > could you rename `api-doc` to `api_doc`?

ok I will


> Why do we need to split this component?

In order to use  this component as client component or they won't work within this Server Components 
read more: https://beta.nextjs.org/docs/rendering/server-and-client-components#moving-client-components-to-the-leaves

File: src/pages/api/signIn.ts (hunk 2 above)
Comment (HoangCacLoanChau) [reply src/pages/api/signIn.ts:7]: ok, I have done the guideline and report to team

File: src/app/api-doc/ReactSwagger.tsx (hunk 1 above)
Comment (congtrung6391) [reply src/app/api-doc/ReactSwagger.tsx]: How's about use client in page.tsx

**Pull request merged**